src/DGtal/shapes/Mesh.ih

```diff
@@ -256,6 +257,20 @@ DGtal::Mesh<TPoint>::getFace(unsigned int i)
 
 template<typename TPoint>
 inline
+TPoint 
+DGtal::Mesh<TPoint>::getFaceBarycenter(unsigned int i) const
+{
+  TPoint c; 
+  MeshFace aFace = getFace(i);
+  for ( auto &j: aFace){
+    c += getVertex(j);
+  }
+  return c/static_cast<typename TPoint::Component>(aFace.size());
```

Comment:
maybe you should make this clear. Or decide that the output of "getbarycenter" is always a "RealPoint" and thus cast to double the denominator